app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -56,8 +56,8 @@ fun PartiesScreen(
 		MVIState.Type.LOADING -> PicoverLoader(modifier = Modifier.fillMaxSize())
 		MVIState.Type.LOADED -> LoadedContent(
 			parties = state.parties,
-			onPartyClick = viewModel::onPartyClick,
-			onFabClick = viewModel::onAddPartyClick,
```

Comment:
That's the overhead of architecture, note how many implication it causes. Check the implementation of, the same goes for unit tests of these functions, it's just redundant.